lib/api/controllers/securityController.js

```diff
@@ -482,7 +503,7 @@ class SecurityController {
     return this.kuzzle.repositories.role.loadRole(request.input.resource._id)
       .then(role => {
         if (!role) {
-          return Promise.reject(new NotFoundError('Cannot update role "' + request.input.resource._id + '": role not found'));
+          return Bluebird.reject(new NotFoundError(`Cannot update role ${request.input.resource._id}: role not found`));
         }
 
         return this.kuzzle.repositories.role.validateAndSaveRole(_.extend(role, request.input.body), {method: 'update'});
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 121 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)